cli/utilities/packages.py

```diff
@@ -0,0 +1,181 @@
+from cli import Cli
+
+
+class Package(Cli):
+    """This module provides CLI interface for yum/dnf operations"""
+
+    def __init__(self, nodes, manager="yum"):
+        super(Package, self).__init__(nodes)
+        self.manager = manager
+
+    def info(self, pkg=None):
+        """show a package information
+
+        Args:
+            pkg (str): package name
+        """
+        cmd = f"{self.manager} info"
+        if pkg:
+            cmd += f" {pkg}"
+
+        stdout, stderr = self.execute(sudo=True, cmd=cmd)
+        if stderr:
+            return False
+        return stdout
+
+    def list(self, pkg=None):
+        """list a package installed
+
+        Args:
+            pkg (str): package name
+        """
+        cmd = f"{self.manager} list "
+        if pkg:
+            cmd += f" {pkg}"
+        stdout, stderr = self.execute(sudo=True, cmd=cmd)
+        if stderr:
+            return False
+        return stdout
+
+    def install(self, pkg):
+        """install a package or packages
+
+        Args:
+            pkg (str): package need to be installed
+        """
+        cmd = f"{self.manager} install -y {pkg}"
+        return self.execute(sudo=True, long_running=True, cmd=cmd)
+
+    def upgrade(self, pkg):
+        """upgrade a package or packages
+
+        Args:
+            pkg (str): package need to be installed
+        """
+        cmd = f"{self.manager} upgrade -y {pkg}"
+        return self.execute(sudo=True, long_running=True, cmd=cmd)
+
+    def add_repo(self, repo, nogpgcheck=False):
+        """enable repos with config-manager command
+
+        Args:
+            repo (str): repo to be enabled
+        """
+        cmd = f"yum-config-manager --add-repo {repo}"
+        if nogpgcheck:
```

Comment:
I don't think we could add a repo and set `--nogpgcheck` at the same time. Please confirm.

This being a utility, I strongly believe we should handle the outcome.